src/components/Page/Page.js

```diff
@@ -68,6 +76,21 @@ export default class Page extends Component {
     )
   }
 
+  findTagDocs (tagHandle, navigation) {
```

Comment:
As discussed, bit tricky, the parser is then making assumptions that the UI can't display tag information together (which we currently can't, as the tags form the navigation).

Secondly the parser is a bit hairy to make this possible right now :(